source/extensions/filters/network/thrift_proxy/conn_manager.cc

```diff
@@ -329,8 +428,13 @@ void ConnectionManager::ActiveRpcDecoderFilter::continueDecoding() {
   }
 }
 
-FilterStatus ConnectionManager::ActiveRpc::applyDecoderFilters(ActiveRpcDecoderFilter* filter) {
-  ASSERT(filter_action_ != nullptr);
```

Comment:
Is this assertion no longer true?

Reply:
Right, the filter_action_ is set in `prepareFilterAction`. Actually it's good to reset to null before we apply filters. Let's me add an opposite assertion.